src/client/ClientSnapRealm.h

```diff
@@ -23,6 +23,8 @@ struct SnapRealm {
 
   SnapRealm *pparent;
   std::set<SnapRealm*> pchildren;
+  utime_t last_modified;
+  uint64_t change_attr;
```

Comment:
This cause failure when running ceph-fuse with valgrind (uninitialized value): https://pulpito.ceph.com/vshankar-2023-02-24_02:11:45-fs-wip-vshankar-testing-20230222.025426-testing-default-smithi/7185976

```
<error>
  <unique>0x0</unique>
  <tid>20</tid>
  <threadname>ms_dispatch</threadname>
  <kind>UninitValue</kind>
  <what>Use of uninitialised value of size 8</what>
  <stack>
    <frame>
      <ip>0x7DA0DDE</ip>
      <obj>/usr/lib64/libstdc++.so.6.0.25</obj>
    </frame>
    <frame>
      <ip>0x7DA193E</ip>
      <obj>/usr/lib64/libstdc++.so.6.0.25</obj>
      <fn>std::ostreambuf_iterator&lt;char, std::char_traits&lt;char&gt; &gt; std::num_put&lt;char, std::ostreambuf_iterator&lt;char, std::char_traits&lt;char&gt; &gt; &gt;::_M_insert_int&lt;unsigned long&gt;(std::ostreambuf_iterator&lt;char, std::char_traits&lt;char&gt; &gt;, std::ios_base&amp;, char, unsigned long) const</fn>
    </frame>
    <frame>
      <ip>0x7DAF508</ip>
      <obj>/usr/lib64/libstdc++.so.6.0.25</obj>
      <fn>std::ostream&amp; std::ostream::_M_insert&lt;unsigned long&gt;(unsigned long)</fn>
    </frame>
    <frame>
      <ip>0x2601E6</ip>
      <obj>/usr/bin/ceph-fuse</obj>
      <fn>operator&lt;&lt;</fn>
      <dir>/opt/rh/gcc-toolset-11/root/usr/include/c++/11</dir>
      <file>ostream</file>
      <line>171</line>
    </frame>
    <frame>
      <ip>0x2601E6</ip>
      <obj>/usr/bin/ceph-fuse</obj>
      <fn>operator&lt;&lt;(std::ostream&amp;, SnapRealm const&amp;)</fn>
      <dir>/usr/src/debug/ceph-18.0.0-2451.g79541ab7.el8.x86_64/src/client</dir>
      <file>ClientSnapRealm.h</file>
      <line>60</line>
    </frame>
    <frame>
      <ip>0x221CEB</ip>
      <obj>/usr/bin/ceph-fuse</obj>
      <fn>Client::update_snap_trace(MetaSession*, ceph::buffer::v15_2_0::list const&amp;, SnapRealm**, bool)</fn>
      <dir>/usr/src/debug/ceph-18.0.0-2451.g79541ab7.el8.x86_64/src/client</dir>
      <file>Client.cc</file>
      <line>5083</line>
```